src/software/thunderscope/common/common_widgets.py

```diff
@@ -382,3 +382,42 @@ def draw_robot(painter, rect, start_angle_degree, span_angle_degree):
     painter.drawChord(
         rect, start_angle_degree * convert_degree, span_angle_degree * convert_degree,
     )
+
+
+def get_string_val(descriptor, value):
```

Comment:
we need this to translate between
- the geneva enum values and other enum values in the parameter tree and the robot status tree
- the motor fault enum values in the motor fault widget

so since we need it in 2 places I thought I'd move it out

Reply:
but isn't it already in the thunderscope/common folder already? it seems like proto_parameter_util is already a shared set of helper functions